package.json

```diff
@@ -1,10 +1,13 @@
 {
   "name": "node-countries",
   "version": "1.6.0",
-  "main": "./index.js",
+  "main": "build/index.js",
   "description": "Countries and Provinces JSON Data",
   "scripts": {
-    "test": "NODE_ENV=test NODE_PATH=./test node ./node_modules/mocha/bin/mocha --exit --harmony "
+    "test": "jest --env=node --colors",
+    "lint": "eslint --ext ts ./",
+    "build": "tsc",
+    "typecheck": "tsc"
```

Comment:
Usually `typecheck` uses `--noEmit` so it only checks types without emitting js files

Reply:
correct done